server/src/hardware/decoder/decoderfunction.cpp

```diff
@@ -90,6 +98,9 @@ void DecoderFunction::worldEvent(WorldState state, WorldEvent event)
   Attributes::setEnabled(name, editable);
   Attributes::setEnabled(type, editable);
   Attributes::setEnabled(function, editable);
+
+  bool momentaryOrHold = (type == DecoderFunctionType::Momentary || type == DecoderFunctionType::Hold);
+  Attributes::setEnabled(timeoutSeconds, editable && momentaryOrHold);
```

Comment:
I prefer to hide properties that don't have effect due to other properties, disabling it might suggest it has effect but can't be changed at the moment. You can use the visible attribute for that.

Reply:
Right. Done

Reply:
I've left the enabled code so it prevents setting values to it.